src/Scripts/Game/UI/EL_BetaHud.c

```diff
@@ -1,3 +1,96 @@
 class EL_BetaHud : SCR_InfoDisplay
 {
-}
+	private SliderWidget staminaSlider = null;
```

Comment:
There is no need to set every method to NULL here as they are NULL by default. Other then that good code.

Reply:
This is changed in new version. See [Update EL_BetaHud.c](https://github.com/EveronLife/EveronLife/pull/78/commits/a34ad014984b170a691f25116b622116c9f63f56)

Additionally I am uploading a change to bring the UI icons lower a bit, it is hard to line up the icons as the ammo one isnt there all the time, but I think Its better now.
Newest update is out [Updated Layout](https://github.com/EveronLife/EveronLife/pull/78/commits/c18300860d78788e743688412432c52903b8119e)